vernac/obligations.ml

```diff
@@ -20,6 +20,12 @@ open Pp
 open CErrors
 open Util
 
+(* EJGA: This should go away, no more need for fix_exn  *)
+module Stm = struct
+    let u = (fun x -> x)
+    let get_fix_exn () = u
+end
+
```

Comment:
I can't find it.  I imagine it is that one Qed in obligations, the last one, has to perform the equivalent of 2 Qeds, since it also declares the actual function (the one using all obligations). Maybe @mattam82 recalls better, I'm not sure at all.